.github/workflows/main.yml

```diff
@@ -211,63 +211,3 @@ jobs:
         with:
           path-to-profile: coverage.txt
           parallel: true
-
```

Comment:
Is the commit message correct? Is it an actual move, or just removal of some tests from github?

Reply:
Also, since this is quite a fundamental change, should we make it its own PR to make sure the CI builds go as expected before starting adding ARM support?

Reply:
Yeah, makes sense. Done: #4482.